snappass/main.py

```diff
@@ -22,13 +24,29 @@
 }
 
 
+def check_redis_alive(fn):
+    def inner(*args, **kwargs):
+        try:
+            redis_client.ping()
+        except ConnectionError as e:
+            print(e)
+            if fn.__name__ == "main":
+                sys.exit(0)
+            else:
+                return abort(500)
```

Comment:
This is good for now, but down the road we may want to try and add a [custom error template](http://flask.pocoo.org/docs/0.11/patterns/errorpages/) with a message explaining the error.